spec/voting_spec.rb

```diff
@@ -59,14 +59,21 @@
       users.each { |user| user.reload }
       expect(users[0].topics_with_vote.pluck(:topic_id)).to contain_exactly(topic1.id)
       expect(users[0].topics_with_archived_vote.pluck(:topic_id)).to contain_exactly()
+
       expect(users[1].topics_with_vote.pluck(:topic_id)).to contain_exactly(topic1.id)
       expect(users[1].topics_with_archived_vote.pluck(:topic_id)).to contain_exactly()
+
       expect(users[2].topics_with_vote.pluck(:topic_id)).to contain_exactly(topic1.id)
       expect(users[2].topics_with_archived_vote.pluck(:topic_id)).to contain_exactly()
+
       expect(users[3].topics_with_vote.pluck(:topic_id)).to contain_exactly()
       expect(users[3].topics_with_archived_vote.pluck(:topic_id)).to contain_exactly()
-      expect(users[4].topics_with_vote.pluck(:topic_id)).to contain_exactly()
-      expect(users[4].topics_with_archived_vote.pluck(:topic_id)).to contain_exactly(topic1.id)
+
+      expect(users[4].topics_with_vote.pluck(:topic_id)).to contain_exactly(topic1.id)
+      expect(users[4].topics_with_archived_vote.pluck(:topic_id)).to contain_exactly()
```

Comment:
Seems a bit odd to me to call `contain_exactly` with nothing. It might be more clear to say `to be_blank`?

Reply:
Ah sure - I was following the existing pattern. I'll make that change. Thanks!